brayns/common/engine/Engine.h

```diff
@@ -98,15 +98,16 @@ class Engine
 
     /**
        Initializes materials for the current scene
-       @param materialType Predefined sets of colors
+       @param colorMap Predefined color map
              MT_DEFAULT: Random colors
```

Comment:
those MT_* values are no longer correct, no?

Reply:
Doc is now in the types.h file. Removed.